src/MOI_wrapper.jl

```diff
@@ -230,6 +249,11 @@ mutable struct Optimizer <: MOI.AbstractOptimizer
     # and FEASIBILITY_SENSE.
     is_feasibility::Bool
 
+    # HiGHS doesn't have special support for binary variables. Cache them here
+    # to modify bounds on solve.
+    binaries::Set{_VariableInfo}
```

Comment:
Then it should not support ZeroOne and rely on the [`ZeroOneBridge`](https://jump.dev/MathOptInterface.jl/dev/submodules/Bridges/reference/#MathOptInterface.Bridges.Constraint.ZeroOneBridge)

Reply:
It's simple enough that I'd rather not rely on the bridge and support them natively.